### PR TITLE
Presa height update

### PR DESCRIPTION
**Pull request by georgebrata**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -10,13 +10,31 @@ const nav = document.querySelector('.nav');
 const tabs = document.querySelectorAll('.operations__tab');
 const tabsContainer = document.querySelector('.operations__tab-container');
 const tabsContent = document.querySelectorAll('.operations__content');
+const slideContainer = document.querySelector('.slider');
+const spinner = document.querySelector('.lds-ellipsis');
+
+console.log(spinner);
```
Comment (georgebrata) [inline script.js]: Please remove the console.log() or comment it. We don't want console logs in production.

File: index.html [hunk 2]
```diff
@@ -1,534 +1,1024 @@
-<!DOCTYPE html>
 <html lang="en">
```
Comment (georgebrata) [inline index.html]: Please update the language to "ro".

File: index.html [hunk 3]
```diff
@@ -1,534 +1,1024 @@
...
+    <header class="header">
+      <nav class="nav">
+        <img
+          src="img/logo.png"
+          alt="Bankist logo"
+          class="nav__logo"
+          id="logo"
+          designer="Joe"
+          data-version-number="3.0"
+          style="opacity: 1"
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class.

File: index.html [hunk 4]
```diff
@@ -1,534 +1,1024 @@
...
+        />
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link" href="#despre" style="opacity: 1">Despre</a>
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class.

File: index.html [hunk 5]
```diff
@@ -1,534 +1,1024 @@
...
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#servicii">Servicii</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--btn btn--show-modal"
+              href="#contact"
+              style="opacity: 1"
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class.

File: index.html [hunk 6]
```diff
@@ -1,534 +1,1024 @@
...
+          <li class="nav__item">
+            <a class="nav__link" href="#servicii" style="opacity: 1"
+              >Servicii</a
+            >
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--btn btn--show-modal"
+              href="#contact"
+              style="opacity: 1"
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class.

File: index.html [hunk 7]
```diff
@@ -1,534 +1,1024 @@
...
-      r.async = 1;
-      r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
-      a.appendChild(r);
-    })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
-  </script>
-</body>
+        <input type="email" />
+        <label for="message">Mesaj</label>
+        <textarea
+          style="resize: none"
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class.

File: index.html [hunk 8]
```diff
@@ -1,534 +1,1024 @@
...
+        <h3 class="section__header">Apariții presă</h3>
+      </div>
 
-      <button class="slider__btn slider__btn--left">&larr;</button>
-      <button class="slider__btn slider__btn--right">&rarr;</button>
-      <div class="dots"></div>
-    </div>
-  </section>
+      <div class="slider">
+        <div class="slide" style="transform: translateX(-800%)">
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.

File: index.html [hunk 9]
```diff
@@ -1,534 +1,1024 @@
...
+                  class="location-link"
+                  href="https://curieruldecluj.ro/sebastian-nagy-tanarul-avocat-care-o-apara-pe-profesoara-ironizata-de-propriii-elevi-in-online-faptele-ilicite-comise-de-catre-parati-sunt-evidente"
+                  target="_blank"
+                  >Curierul de Cluj</a
+                >
+              </p>
+            </address>
+          </div>
+        </div>
+        <div class="slide" style="transform: translateX(-700%)">
```
Comment (georgebrata) [inline index.html]: Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.

**Review by georgebrata — COMMENTED** [review]
Please implement the fixes I suggested.

File: index.html (hunk 9 above)
Comment (adonisjoe) [reply index.html]: I removed the section because we are rendering dynamically from the server

File: index.html [hunk 10]
```diff
@@ -1,534 +1,1010 @@
-<!DOCTYPE html>
```
Comment (georgebrata) [inline index.html:1]: Why did you remove `<!DOCTYPE html>`? This is mandatory for HTML5.

File: index.html [hunk 11]
```diff
@@ -1,534 +1,1010 @@
...
-    href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,500;0,600;0,700;0,800;0,900;1,500;1,600&display=swap"
-    rel="stylesheet" />
-  <link rel="stylesheet" href="style.css" />
-  <link rel="stylesheet" href="queries.css" />
-  <title>Sebastian Gabriel Nagy | Avocat Cluj</title>
+    <script defer="" src="script.js"></script>
+    <script
+      async=""
+      src="https://static.hotjar.com/c/hotjar-3665471.js?sv=6"
+    ></script>
```
Comment (georgebrata) [inline index.html:23]: Why did you remove the HOTJAR script? Please put it back:

```<!-- Hotjar Tracking Code for https://avocat-nagy-cluj.ro -->
<script>
    (function(h,o,t,j,a,r){
        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
        h._hjSettings={hjid:3665471,hjsv:6};
        a=o.getElementsByTagName('head')[0];
        r=o.createElement('script');r.async=1;
        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
        a.appendChild(r);
    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
</script>```

File: index.html [hunk 12]
```diff
@@ -1,534 +1,1010 @@
...
-      <!--TODO: smaller image, rounded borders -->
-      <img src="img/hero.jpg" class="header__img"
-        alt="Avocat Sebastian Nagy imagine profil | Avocat definitiv Cluj-Napoca" />
-    </div>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <img
+          src="img/logo.png"
+          alt="Bankist logo"
```
Comment (georgebrata) [inline index.html]: Please add a valid alt attribute for this image. A good one should be "Sebastian Gabriel Nagy | Avocat Cluj logo".

File: index.html [hunk 13]
```diff
@@ -1,534 +1,1010 @@
...
+            <a class="nav__link" href="#servicii">Servicii</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__link--btn btn--show-modal" href="#contact"
+              >Contact</a
+            >
+          </li>
+        </ul>
+        <button id="toggleButton-close">
+          <svg
```
Comment (georgebrata) [inline index.html]: Please move the .svg files in a separate file and use them with  in the HTML so we don't pollute the HTML with tons on useless lines of svg code.

File: index.html [hunk 14]
```diff
@@ -1,534 +1,1010 @@
...
+              Activitatea noastră este caracterizata de inovare și
+              adaptabilitate la nevoile clientului, transparență, atenție la
+              detalii, experiență și expertiza de înaltă calitate și nu în
+              ultimul rand, disponibilitate în relația cu clienții.
+            </span>
+          </p>
+        </div>
         <!-- <button class="btn--text btn--scroll-to">Mai mult &DownArrow;</button> -->
-        <a class="btn--show-modal" href="#contact">Contactează</a>
+        <button class="btn--show-modal btn--text">Contactează</button>
```
Comment (georgebrata) [inline index.html]: This button should scroll to the bottom of the website, not open the modal. Please put back the `href` attribute.

File: index.html [hunk 15]
```diff
@@ -1,534 +1,1010 @@
...
-      </div>
-
-      <div class="features__feature">
-        <div class="features__icon">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-trending-up"></use>
-          </svg>
+        <div class="operations__content operations__content--1">
+          <div class="operations__icon operations__icon--1">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
```
Comment (georgebrata) [inline index.html]: Please extract the svg in a separate file and use it with the  tag in HTML.

File: index.html [hunk 16]
```diff
@@ -1,534 +1,1010 @@
...
-      <img src="img/law-1--lazy.jpg" data-src="img/law-1.jpg" alt="Computer"
-        class="features__img features__img-m lazy-img" />
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-monitor"></use>
-          </svg>
+        <div class="operations__content operations__content--2">
+          <div class="operations__icon operations__icon--2">
+            <svg
```
Comment (georgebrata) [inline index.html]: Same as above.

File: index.html [hunk 17]
```diff
@@ -1,534 +1,1010 @@
...
+            dezbate conceptele juridice într-un mod accesibil pentru elevi.
+          </p>
         </div>
-        <div class="features__deatail-cont">
-          <h5 class="features__header">Cabinetul oferă servicii juridice</h5>
+        <div
+          class="operations__content operations__content--3 operations__content--active"
+        >
+          <div class="operations__icon operations__icon--3">
+            <svg
```
Comment (georgebrata) [inline index.html]: Same as above.

File: index.html [hunk 18]
```diff
@@ -1,534 +1,1010 @@
...
+        <h3 class="section__header">
+          Servicii juridice de încredere în Cluj-Napoca
+        </h3>
+      </div>
+
+      <div class="features">
+        <img
+          src="img/law-1.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
```
Comment (georgebrata) [inline index.html]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii".

File: index.html [hunk 19]
```diff
@@ -1,534 +1,1010 @@
...
+          <p>
+            Soluționarea problemelor legate de divorț, custodie, adopție și alte
+            aspecte legate de dreptul familiei.
+          </p>
         </div>
-        <div class="features__deatail-cont">
+        <img
+          src="img/law-2.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
```
Comment (georgebrata) [inline index.html]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale".

File: index.html [hunk 20]
```diff
@@ -1,534 +1,1010 @@
...
+          class="features__img"
+        />
+
+        <img
+          src="img/law-3.jpg"
+          data-src="img/law-3.jpg"
+          alt="Credit card"
```
Comment (georgebrata) [inline index.html]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii juridice".

File: index.html [hunk 21]
```diff
@@ -1,534 +1,1010 @@
...
           </p>
         </div>
       </div>
-    </div>
-  </section>
+      <div class="features features-m">
+        <img
+          src="img/law-1--lazy.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
```
Comment (georgebrata) [inline index.html]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale si juridice".

File: index.html [hunk 22]
```diff
@@ -1,534 +1,1010 @@
...
+              Avocatul Sebastian Nagy Cabinetul oferă servicii juridice în
+              diverse domenii de drept, adaptate nevoilor specifice
+              dumneavoastră.
+            </p>
+          </div>
+        </div>
+        <img
+          src="img/law-3--lazy.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
```
Comment (georgebrata) [inline index.html:336]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale".

File: index.html [hunk 23]
```diff
@@ -1,534 +1,1010 @@
...
+              Apărare și reprezentare în dosare penale, asigurându-se că
+              drepturile și interesele clienților săi sunt protejate cu
+              fermitate
+            </p>
+          </div>
+        </div>
+        <img
+          src="img/law-3--lazy.jpg"
+          data-src="img/law-3.jpg"
+          alt="Credit card"
```
Comment (georgebrata) [inline index.html]: Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii juridice".

File: index.html [hunk 24]
```diff
@@ -1,534 +1,1010 @@
...
-        <button class="btn operations__tab operations__tab--3" data-tab="3">
-          Pasiune
-        </button>
-      </div>
-      <div class="operations__content operations__content--1 operations__content--active">
-        <div class="operations__icon operations__icon--1">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-upload"></use>
-          </svg>
+      <div class="header__title header__title-m">
```
Comment (georgebrata) [inline index.html:111]: Do you close the `</header>` element anywhere? It seems like you removed some closing tags and this might introduce bugs, please double-check.

**Review by georgebrata — COMMENTED** [review]
Please fix
